hack/update/generate-crds.sh

```diff
@@ -51,7 +51,9 @@ echo "Generating OpenAPI v3 schemas for chart CRDs"
 bin/controller-gen \
   schemapatch:manifests="${CRD_DIR}" \
   output:dir="${CRD_DIR}" \
+  crd:crdVersions=v1 \
   paths=./pkg/apis/...
+mv ${CRD_DIR}/helm.fluxcd.io_helmreleases.yaml ${CRD_DIR}/helmrelease.yaml
```

Comment:
This is a weird behavior that the file will be different when the crd version is set. I do not know why this happens and I could not figure out how to fix it.

Reply:
I think an option may be to use `output:stdout` (instead of `output:dir=...`) and then write it directly to the file, instead of moving one around.

Reply:
Thanks for the tip, makes it a lot cleaner. I did however have to trim the first two lines of the generated CRD as the new line and `---`caused other issues.